packages/utils/telemetry-utils/src/logger.ts

```diff
@@ -224,6 +227,13 @@ export abstract class TelemetryLogger implements ITelemetryLogger {
                 }
             }
         }
+
+        // Do one more pass over the properties to handle any tagged information:
+        if (eventCanSupportTags === undefined || eventCanSupportTags === false) {
+            for (const key of Object.keys(newEvent)) {
```

Comment:
Nit -- Consider putting this for loop inside `redactTaggedUserData`?

Reply:
Sure, wasn't sure whether to make that method atomic or not

Reply:
Sure, either way